### PR TITLE
Feature/ca engine 201

### LINKED ISSUE #127: HTTP proxy support for CMC client
When using CA CMC it should be possible to configure a HTTP proxy.

### PR DESCRIPTION
**Pull request by Razumain**
Updating to latest version of ca-engine and ca-cmc.
Providing means to configure HTTP proxy for CMC client requests.

Resolves #127

### CODE REVIEW

File: keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java [hunk 1]
```diff
@@ -69,6 +69,20 @@ public class CMCKeyAndCertificateHandlerConfiguration extends AbstractKeyAndCert
   @Setter
   private RemoteCaInformation remoteCaInfo;
 
+  /**
```
Comment (martin-lindstrom) [inline keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java:72]: I suggest making a HttpProxy object instead. As it is now we can't configure user name or password (should it be needed).

Comment (martin-lindstrom) [reply keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java:72]: Compare with how we solved it for SAML metadata.

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Missing updates of version numbers in POM.
Also, missing documentation changes for changed config in markdown file.

File: keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java (hunk 1 above)
Comment (Razumain) [reply keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java:75]: Comments should be resolved in last update

**Pull request merged**